content/en/docs/reference/scheduling/config.md

```diff
@@ -255,10 +226,30 @@ the same configuration parameters (if applicable). This is because the scheduler
 only has one pending pods queue.
 {{< /note >}}
 
+## Migration introductions between api versions
+
+### v1beta2 -> v1beta3
+* Some plugins' weight has been modified for user preferences:
+  * `InterPodAffinity` to 2
+  * `NodeAffinity` to 2
+  * `TaintToleration` to 3
+* Support for configuration file version `v1beta1` is removed.
+* Scheduler plugin `ServiceAffinity` is removed.
+* The legacy scheduler policy config is removed, the associated flags `policy-config-file`, `policy-configmap`, `policy-configmap-namespace` and `use-legacy-policy-config` are also removed.
+
+### v1beta1 -> v1beta2
```

Comment:
```suggestion
### Scheduler configuration migration: v1beta1 → v1beta2 {#migration-v1beta1-v1beta2}
```

I would order this _before_ the migration from v1beta2 to v1beta3.

Reply:
I would use the tabs as advised.